dev_guide/builds/triggering_builds.adoc

```diff
@@ -436,6 +436,22 @@ strategy:
 This ensures that the triggered build uses the new image that was just pushed to
 the repository, and the build can be re-run any time with the same inputs.
 
+An image change trigger can be paused allowing for multiple changes on referenced
```

Comment:
on referenced -> on the referenced

I would put a comma after "... can be paused"